src/transformers/models/rwkv/modeling_rwkv.py

```diff
@@ -623,7 +623,16 @@ def forward(
         output_attentions: Optional[bool] = None,
         output_hidden_states: Optional[bool] = None,
         return_dict: Optional[bool] = None,
+        attention_mask: Optional[bool] = None,
     ) -> Union[Tuple, RwkvOutput]:
+        if attention_mask is not None:
+            logger.warning_once(
+                "`RwkvModel` doesn't use `attention_mask` but its `forward` method receives a value which is not "
+                "`None`. The inputs are likely prepared with a tokenizer that outputs `attention_mask`. This will be "
+                "discarded by setting the value to `None`."
+            )
+            attention_mask = None  # noqa
```

Comment:
I wouldn't set any warning here. Just document that this argument is not used yet (we will support it in the future but it needs extra code).

Reply:
OK!